app/src/main/res/drawable-xxhdpi/ic_personal_request.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_personal_request` appears to be unused
 Unused resources make applications larger and slow down builds.